components/header/Header.vue

```diff
@@ -42,145 +42,89 @@
               <sparc-logo />
             </nuxt-link>
           </div>
-          <button
-            class="nav-main-container__mobile-search"
-            @click="openMobileSearch"
-            @enter="executeSearch(searchQuery)"
-          >
-            <svg-icon
-              icon="icon-magnifying-glass"
-              height="25"
-              width="25"
-              dir="left"
-            />
-          </button>
           <div :class="[mobileSearchOpen ? 'search-overlay' : '']">
```

Comment:
If we are removing the search from the header why do we need to know if it is open or not?

Reply:
You're right @hsorby  we don't. I was asked to change as little code as possible since we could be bringing it back, but I am happy to remove the overlay elements if you like

Reply:
Yeah I think removing it all is fine and maybe just link the PR on the ticket for the future if we end up adding it back (even though we could also just sift through the git history)